### DISCUSSION TITLE
[Feature Request] when  Can Support Foundry Tool?

### DISCUSSION THREAD

**Discussion by JohnLanPeluchi** [zkSync-Community-Hub/zksync-developers#361] [Feature requests]
### Network

Both

### Provide a brief description of the feature you would like to see implemented.

hope support foundry tool 

### Who does this feature request benefit the most (users, developers, protocol, etc.)?

developers

### Additional Details

not more

**Comment by dutterbutter** [comment]
Hey @JohnLanPeluchi you can try out `foundry-zksync` here: https://github.com/matter-labs/foundry-zksync

**Comment by gregmolina5** [comment]
Implementing support for the Foundry tool would greatly benefit developers by enhancing their development workflow and productivity.

**Reply by dutterbutter** [reply]
See above comment.
